exercises/nodejs/src/game.js

```diff
@@ -1,172 +1,206 @@
-        }
+        console.log(playerNames[currentPlayer] + ' is getting out of the penalty box')
+        movePlayerPosition(roll)
 
-        console.log(players[currentPlayer] + "'s new location is " + places[currentPlayer])
-        console.log('The category is ' + currentCategory())
-        askQuestion()
+        askQuestion(currentCategory())
       } else {
-        console.log(players[currentPlayer] + ' is not getting out of the penalty box')
+        console.log(playerNames[currentPlayer] + ' is not getting out of the penalty box')
         isGettingOutOfPenaltyBox = false
       }
     } else {
-      places[currentPlayer] = places[currentPlayer] + roll
-      if (places[currentPlayer] > 11) {
-        places[currentPlayer] = places[currentPlayer] - 12
-      }
+      movePlayerPosition(roll)
 
-      console.log(players[currentPlayer] + "'s new location is " + places[currentPlayer])
-      console.log('The category is ' + currentCategory())
-      askQuestion()
+      askQuestion(currentCategory())
     }
   }
 
-  this.wasCorrectlyAnswered = function () {
+  this.questionAnsweredCorrectly = function () {
     if (inPenaltyBox[currentPlayer]) {
       if (isGettingOutOfPenaltyBox) {
         console.log('Answer was correct!!!!')
-        purses[currentPlayer] += 1
-        console.log(players[currentPlayer] + ' now has ' +
-            purses[currentPlayer] + ' Gold Coins.')
+        increasePurse()
 
         var winner = didPlayerWin()
-        currentPlayer += 1
-        if (currentPlayer == players.length) { currentPlayer = 0 }
+        nextPlayer()
 
         return winner
       } else {
-        currentPlayer += 1
-        if (currentPlayer == players.length) { currentPlayer = 0 }
+        nextPlayer()
         return true
       }
     } else {
-      console.log('Answer was correct!!!!')
 
-      purses[currentPlayer] += 1
-      console.log(players[currentPlayer] + ' now has ' +
-          purses[currentPlayer] + ' Gold Coins.')
+      console.log('Answer was correct!!!!')
+      increasePurse()
 
       var winner = didPlayerWin()
 
-      currentPlayer += 1
-      if (currentPlayer == players.length) { currentPlayer = 0 }
-
+      nextPlayer()
       return winner
     }
   }
 
-  this.wrongAnswer = function () {
+  function increasePurse() {
+    playerPurses[currentPlayer] += 1
+    console.log(playerNames[currentPlayer] + ' now has ' + playerPurses[currentPlayer] + ' Gold Coins.')
+  }
+  
+  this.questionAnsweredIncorrectly = function () {
     console.log('Question was incorrectly answered')
-    console.log(players[currentPlayer] + ' was sent to the penalty box')
+    console.log(playerNames[currentPlayer] + ' was sent to the penalty box')
     inPenaltyBox[currentPlayer] = true
 
-    currentPlayer += 1
-    if (currentPlayer == players.length) { currentPlayer = 0 }
+    nextPlayer()
     return true
   }
+
+  function nextPlayer() {
+    currentPlayer = (currentPlayer + 1) % playerNames.length;
+  }
```

Comment:
This `nextPlayer` function makes good use of the `%` operator.
Also it was a good refactor for reducing duplicated code.